pgjdbc/src/test/java/org/postgresql/test/jdbc2/StringTypeUnspecifiedArrayTest.java

```diff
@@ -42,6 +45,9 @@ protected void updateProperties(Properties props) {
   @Test
   public void testCreateArrayWithNonCachedType() throws Exception {
     PGbox[] in = new PGbox[0];
+    TypeInfo typeInfo = ((PgConnection)con).getTypeInfo();
+    // Previously we put didn't cache this. now we do. Remove it for the test
+    typeInfo.invalidate("box[]");
```

Comment:
I incline we should create a brand new type instead of relying on invalidate.

Reply:
Yes, that would be the best way to do this. I'll spend some time on that today

Reply:
I suggest we make a one-line change that adds box, release 42.5, and add tests later